src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/AssetSchedules/AssetSchedule.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Volo.Abp.Domain.Entities.Auditing;
+using Volo.Abp.MultiTenancy;
+
+namespace EasyAbp.BookingService.AssetSchedules;
+
+public class AssetSchedule : FullAuditedAggregateRoot<Guid>, IHasPeriodInfo, IMultiTenant
+{
+    public virtual Guid? TenantId { get; protected set; }
+    
+    public virtual Guid AssetId { get; protected set; }
+    
+    public virtual DateTime Date { get; protected set; }
+
+    public virtual TimeSpan StartingTime { get; protected set; }
+    
+    public virtual TimeSpan Duration { get; protected set; }
+    
+    /// <summary>
+    /// Accept or reject occupying within this time frame.
+    /// </summary>
+    public virtual AssetSchedulePolicy SchedulePolicy { get; protected set; }
+
```

Comment:
The `AssetSchedule` should have `AdvanceBookingDays`  property too.
When it is null, fall back to `AssetCategory`.